test/Concurrency/taskgroup_cancellAll_from_parent.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-typecheck-verify-swift  -disable-availability-checking
```

Comment:
test file does not align with what the test does: this is `taskgroup_cancelAll_from_child.swift` really -- which we now ban.

Reply:
Agreed. Will rename.